### PR TITLE
Added the installer plugin

### PR DESCRIPTION
**Pull request by hemna**
This plugin creates 2 aliases "in and up" that are system
and distro agnostic.   For example,
on Mac in = brew install
on SUSE in = sudo -E zypper install
on Debian in = sudo -E apt-get install
on RedHat in = sudo -E yum install

add the 'installer' to your plugin list in ~/.zshrc
source ~/.zshrc

now you can run
'in wget'
'up curl'

### CODE REVIEW

File: plugins/installer/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+On Mac systems, it's assumed the installer is brew
+On Suse based sysetms, it's zypper
+On Redhat, it's yum
+On Debian/Ubuntu it's apt-get
+
+include the installer as a plugin in your ~/.zshrc
+
+to install a new app run
+install <app>
```
[the remark this answers is not in the document]
Comment (hemna) [reply plugins/installer/README.md]: yes

File: plugins/installer/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+you can upgrade an app with
+upgrade <app>
```
[the remark this answers is not in the document]
Comment (hemna) [reply plugins/installer/README.md]: yes

**Comment by hemna** [comment]
I'm trying to figure out how to push up a new version of this to make this review happy

**Comment by hemna** [comment]
Any updates on this review?   it's been sitting a while.

**Comment by mcornella** [comment]
In the end this is not a feasible plugin to have:

- Most package managers have different package names, it would cause confusion if a command worked in one package manager and not in another.
- Some package managers exist for the same operating system or distribution, and knowing which to pick would be difficult.
- I expect a lot of changes to happen for this plugin because of differences to different package managers, new distributions to support, etc., adding significant maintenance toil with difficulty to test contributions.

Thank you for the contribution.

**Pull request closed without merging**